### PR TITLE
Crashlytics v4.0.0-beta.3 CHANGELOG

### PR DESCRIPTION
**Pull request by samedson**

### CODE REVIEW

File: Crashlytics/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# v4.0.0-beta.3
+
+- [fixed] Fixed symbol collisions with the `Crashlytics` SDK and added a warning not to include both (#4753, #4755)
```
Comment (RelaxedPear) [inline Crashlytics/CHANGELOG.md]: Let's change "`Crashlytics` SDK" to "Firebase Crashlytics SDK" (without code font)

Comment (samedson) [reply Crashlytics/CHANGELOG.md]: This is referring to the old Crashlytics SDK - how should we refer to that?

Comment (morganchen12) [reply Crashlytics/CHANGELOG.md]: maybe "Fabric Crashlytics SDK"?

Comment (RelaxedPear) [reply Crashlytics/CHANGELOG.md]: Oh oops, just read this thread. Let's change this to "...with the legacy Fabric SDK".

**Pull request merged**